src/components/editor/YextEntityFieldSelector.tsx

```diff
@@ -33,6 +34,7 @@ const TYPE_TO_CONSTANT_CONFIG: Record<string, Field<any>> = {
   "type.phone": TEXT_CONSTANT_CONFIG,
   "type.image": IMAGE_CONSTANT_CONFIG,
   "type.address": ADDRESS_CONSTANT_CONFIG,
+  c_cta: CTA_CONSTANT_CONFIG,
```

Comment:
Also not you, but the starter has a custom CTA type which is link and label, but I think we're supposed to be supporting the built-in Call To Action type which is link and label, but link has a type of URL/Phone/Email/Other

Reply:
I'll update to use the built-in type in another PR

Reply:
Sounds good. I'm not sure we want to hardcode anything for c_cta though because you could create your own custom type called c_cta and it be completely different

Reply:
Updated to use type.cta